.gitignore

```diff
@@ -4,3 +4,4 @@ phpunit.xml
 composer.lock
 composer.phar
 vendor/*
+.idea/*
```

Comment:
This should not be added to the project ignore file.

Reply:
This has been reverted by https://github.com/KnpLabs/php-github-api/pull/860/commits/7742bd1d322b4129ac56ece532ae2c45cd9d6738